x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/use_create_analytics_form.ts

```diff
@@ -39,12 +39,25 @@ export interface CreateAnalyticsFormProps {
   state: State;
 }
 
+interface ErrorResponse {
```

Comment:
question: do we have a generic interface for the error response?

Reply:
Added new `ErrorResponse` interface in https://github.com/elastic/kibana/pull/60249/commits/d8926e25a3a4e1c7e3dd20a6342c15cba035bfc5